docs/development/dev_guide.md

```diff
@@ -37,10 +37,10 @@ To use the dev containers for GraphScope development, you can follow these steps
 To install all dependencies on your local, use the GraphScope command-line utility [gs](https://github.com/alibaba/GraphScope/blob/main/gs) with the subcommand `install-deps` like this
 
 ```bash
-./gs install-deps dev
+gsctl install-deps dev
```

Comment:
By using `gsctl xxx` directly, you are assuming the `gsctl` could be found in the `PATH`. Either you should add a step to install `gsctl` to the system path, or you need to add the path to gsctl, for example: `./gsctl xxx`.

This applies to all other `gsctl` usages throughout the documents.